src/pow.cpp

```diff
@@ -401,13 +402,9 @@ RCP<const Basic> pow_expand(const RCP<const Pow> &self)
                         pownum(i2->second,
                             rcp_static_cast<const Number>(exp)));
                     } else if (is_a<Complex>(*(i2->second))) {
-                        if (is_a<Integer>(*exp)) {
-                            RCP<const Integer> pow_new = rcp_static_cast<const Integer>(exp);
-                            RCP<const Complex> base_new = rcp_static_cast<const Complex>(i2->second);
-                            pow_complex(outArg(overall_coeff), base_new, *pow_new);
```

Comment:
`overall_coeff` needs to be mulitplied by the result of `pow_complex`, not set to result of `pow_complex`